packages/api/src/codegen/codegenerator.ts

```diff
@@ -28,8 +28,9 @@ export default abstract class CodeGenerator {
   requiredPackages!: Record<
     string,
     {
+      dependencyType: 'production' | 'development';
```

Comment:
Made this as `dependencyType` instead of `devDependency: boolean` so it's not specific to NPM.

Reply:
yeah that's a great call